tests/asyncapi-2.0/Message Object/Payloads/includes/user-datatype.raml

```diff
@@ -0,0 +1,6 @@
+#%RAML 1.0 DataType
+
+properties:
+  username:
+    type: string
```

Comment:
`type: string` and `required: true` are implicit in RAML so I would say let's remove those lines

Reply:
Done